platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheIO.kt

```diff
@@ -245,8 +251,17 @@ class ConfigurationCacheIO internal constructor(
      * @param profile the unique name associated with the output stream for debugging space usage issues
      */
     internal
-    fun writerContextFor(outputStream: OutputStream, profile: () -> String): Pair<DefaultWriteContext, Codecs> =
-        KryoBackedEncoder(outputStream).let { encoder ->
+    fun writerContextFor(outputStream: OutputStream, parallelize: Boolean, profile: () -> String): Pair<DefaultWriteContext, Codecs> =
+        (when {
+            parallelize -> KryoBackedEncoder(
+                ParallelOutputStream.of { FramedSnappyCompressorOutputStream(outputStream) },
```

Comment:
❓ Have you tried using `tunedForSpeed()` here?

Reply:
I have not, good idea 👍